### DISCUSSION TITLE
Erro o disfunción de un mod

### DISCUSSION THREAD

**Discussion by GokuBlack177** [thatskymod/Sky-CotL-Scripts#141] [Q&A]
hay un mod que no me deja entrar y los otros 2 no me funcionan, el mod que no me deja

**Comment by thatskymod** [comment]
El problema se origina en la biblioteca libLangit, que se volvió incompatible después de la última actualización del juego. He marcado la biblioteca como incompatible y he publicado una nueva versión, [libLangit 0.4.4](https://github.com/thatskymod/Sky-CotL-Scripts/blob/05fa329f2e50de2d33b6232b5005705f45e5ac91/Canvas/Libs/libLangit044.so), que debería solucionar los fallos inmediatos.

Note que fechar seu próprio tópico não é uma boa ideia, pois pode torná-lo inacessível a outros leitores. Isso é especialmente importante se você precisar de mais ajuda ou se ainda for necessária alguma contribuição.

<details>
  <summary>Show English version</summary>
  
  The issue originates from the libLangit library, which became incompatible after the latest game update. I have flagged the library as incompatible and released a new version, [libLangit 0.4.4](https://github.com/thatskymod/Sky-CotL-Scripts/blob/05fa329f2e50de2d33b6232b5005705f45e5ac91/Canvas/Libs/libLangit044.so), which should fix the immediate crashes.

Note that closing your own thread is not a good idea, as it may make it inaccessible to other readers. This is especially if you require further help or input is still needed.
  
</details>
